tools/windows-node-installer/README.md

```diff
@@ -24,86 +24,43 @@ make build-tools
 ```
 
 ## How to use it
-### Creating a Windows instance:
-
-```bash
-wni --help
-Creates and destroys Windows instance with an existing OpenShift cluster.
-
-Usage:
-  wni [command]
 
-Available Commands:
-  aws         Takes aws specific resource names from user
-  help        Help about any command
+The `wni` requires resources including the kubeconfig of the OpenShift cluster, a provider specific credentials file 
+and account to create and destroy a Windows instance on the selected provider. To create an instance, `wni` also 
+requires image id, instance type, and ssh key. Some optional flags include directory path to 
+windows-node-installer.json file and log level display.
 
-Flags:
-      --dir string          directory to save or read window-node-installer.json file from. (default ".")
-  -h, --help                help for wni
-      --kubeconfig string   file path to the kubeconfig of the existing OpenShift cluster (required).
-      --log-level string    log level (e.g. 'info') (default "info")
+The following commandline example creates and destroys a Windows instance on AWS platform. For more information please 
```

Comment:
Please remove `The following commandline example creates and destroys a Windows instance on AWS platform.` Add the `For more information...` line to the previous paragraph.